### PR TITLE
solution was added

### PR DESCRIPTION
**Pull request by Maksym845**
- [DEMO LINK](https://Maksym845.github.io/react_goods-selector/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +15,84 @@ export const goods = [
...
-          </td>
+        <table className="table">
+          <tbody>
+            {goods.map(item => {
+              return (
+                <tr
+                  data-cy="Good"
+                  className={selectedGood === item
+                    ? 'has-background-success-light'
+                    : ''}
```
Comment (Yeliseiev) [inline src/App.tsx:62]: I think it`s better to use classnames libary here.

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +15,84 @@ export const goods = [
...
+                >
+                  <td>
+                    <button
+                      data-cy={selectedGood === item
```
Comment (Yeliseiev) [inline src/App.tsx:66]: You use this expression quite often. Maybe you can move it to a separate variable.

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +15,84 @@ export const goods = [
...
+                      type="button"
+                      className={selectedGood === item
+                        ? 'button is-info'
+                        : 'button'}
+                      onClick={() => {
+                        if (selectedGood === item) {
+                          this.setState({ selectedGood: '' });
+                        } else {
+                          this.setState({ selectedGood: item });
+                        }
```
Comment (Yeliseiev) [inline src/App.tsx:78]: Use ternary operator here

**Review by Yeliseiev — APPROVED** [review]
Good job!!

File: src/App.tsx [hunk 4]
```diff
@@ -15,71 +15,84 @@ export const goods = [
...
-          </td>
+    return (
+      <main className="section container">
+        {selectedGood === ''
+          ? (
+            <h1 className="title">
+              No goods selected
+            </h1>
+          )
+          : null}
```
Comment (Andr1yK) [inline src/App.tsx:38]:
```suggestion
        {!selectedGood && (
            <h1 className="title">
              No goods selected
            </h1>
        )}
```

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**